src/bebb/main.mo

```diff
@@ -148,6 +148,25 @@ actor {
     };
   };
 
+  /**
+   * Public interface for finding matching Entities according to supported search parameters. Retrieving an Entity is subject to any permission and rules defined by the Entity.
+   * Any matching Entities found are returned, otherwise an empty array is returned.
+  */
+  public shared query ({ caller }) func match_entities(filterCriteria : [Entity.EntityFilterCriterion]) : async Entity.EntitiesResult {
+    if (filterCriteria.size() < 1) {
+      return #Err(#Unauthorized "Must specify matchCriteria");
+    };
+    if (filterCriteria.size() > 5) {
```

Comment:
I would make 5 a variable and then use the variable here. Then you can document the variable and it is more clear on how to change it